src/Python/Visualization/open3d_visualizer.cpp

```diff
@@ -43,11 +43,11 @@ void pybind_visualizer(py::module &m)
         .def("__repr__", [](const Visualizer &vis) {
             return std::string("Visualizer with name ") + vis.GetWindowName();
         })
-        .def("create_window", &Visualizer::CreateWindow,
+        .def("create_window", &Visualizer::CreateVisualizerWindow,
```

Comment:
Shouldn't we also need to change ``create_windows`` as well? Every Python function names are lower case + under bar style of original c++ function name.

Reply:
I thought about it but decided against it. This would look like a weird name in Python interface.